source/src/drivers/cs_Watchdog.cpp

```diff
@@ -15,19 +15,22 @@
 // ============== Static members ==============
 nrfx_wdt_channel_id Watchdog::_channelId;
 
+/*
+ * Can be used to write information towards the bootloader or the app across a warm boot.
+ */
+void handleTimeout() {
+}
 
 void Watchdog::init() {
 	nrfx_wdt_config_t config;
 	config.behaviour = (nrf_wdt_behaviour_t)NRFX_WDT_CONFIG_BEHAVIOUR;
 	config.reload_value = CS_WATCHDOG_TIMEOUT_MS;
-//	config.interrupt_priority = CS_WATCHDOG_PRIORITY;
-
+	config.interrupt_priority = CS_WATCHDOG_PRIORITY;
 	LOGi("Init watchdog with %u ms timeout", CS_WATCHDOG_TIMEOUT_MS);
 
 	uint32_t err_code;
 
-//	err_code = nrfx_wdt_init(&config, &handleTimeout);
-	err_code = nrfx_wdt_init(&config, NULL);
+	err_code = nrfx_wdt_init(&config, &handleTimeout);
```

Comment:
Why do we set an interrupt while we're not using it?

Reply:
See explanation last commit